pokesets/Manga/Pokemon Adventures/PS-Crystal.yaml

```diff
@@ -0,0 +1,245 @@
+species: Smoochum
+setname: PS-Crystal
+ingamename: Chumee
+displayname: Chumee
+ability: [Oblivious]
+item: [null]
+ball: [Poke]
+moves:
+- [Mean Look]
+- [Icy Wind]
+gender: [f]
+happiness: 255
+biddable: true
+rarity: 1.0
+nature: Calm
+evs: {hp: 252, atk: 0, def: 0, spA: 4, spD: 252, spe: 0}
+ivs: 31
+tags: [PS-Crystal, manga]
+---
+species: Parasect
+setname: PS-Crystal
+ingamename: Parasee
+displayname: Parasee
+ability: [Dry Skin]
+item: [null]
+ball: [Poke]
+moves:
+- [Spore]
+- [Sweet Scent]
+- [Giga Drain]
+- [Sludge Bomb]
+gender: [f]
+happiness: 255
+biddable: true
+rarity: 1.0
+nature: Brave
+evs: {hp: 248, atk: 0, def: 0, spA: 252, spD: 8, spe: 0}
+ivs: 31
+tags: [PS-Crystal, manga]
+---
+species: Cubone
+setname: PS-Crystal
+ingamename: Bonee
+displayname: Bonee
+ability: [Lightning Rod]
+item: [null]
+ball: [Poke]
+moves:
+- [Bonemerang]
+- [Bone Club]
+- [False Swipe]
+- [Bone Rush]
+gender: [m]
+happiness: 255
+biddable: true
+rarity: 1.0
+nature: Lonely
+evs: {hp: 248, atk: 252, def: 0, spA: 0, spD: 8, spe: 0}
+ivs: 31
+tags: [PS-Crystal, manga]
+---
+species: Hitmonchan
+setname: PS-Crystal
+ingamename: Monlee
+displayname: Monlee
+ability: [Iron Fist]
+item: [null]
+ball: [Poke]
+moves:
+- [Bonemerang]
+- [Bone Club]
+- [False Swipe]
+- [Bone Rush]
+gender: [m]
+happiness: 255
+biddable: true
+rarity: 1.0
+nature: Adamant
+evs: {hp: 0, atk: 252, def: 0, spA: 0, spD: 4, spe: 252}
+ivs: 31
+tags: [PS-Crystal, manga]
+---
+species: Xatu
+setname: PS-Crystal
+ingamename: Natee
+displayname: Natee
+ability: [Early Bird]
+item: [null]
+ball: [Poke]
+moves:
+- [Psychic]
+- [Fly]
+- [Future Sight]
+- [Tailwind]
+gender: [f]
+happiness: 255
+biddable: true
+rarity: 1.0
+nature: Quiet
+evs: {hp: 0, atk: 4, def: 0, spA: 252, spD: 0, spe: 252}
+ivs: 31
+tags: [PS-Crystal, manga]
+---
+species: Arcanine
+setname: PS-Crystal
+ingamename: Archy
+displayname: Archy
+ability: [Intimidate]
+item: [null]
+ball: [Poke]
+moves:
+- [Extreme Speed]
+- [Bite]
+- [Flame Wheel]
+- [Flamethrower]
+gender: [m]
+happiness: 255
+biddable: true
+rarity: 1.0
+nature: Hasty
+evs: {hp: 0, atk: 252, def: 0, spA: 4, spD: 0, spe: 252}
+ivs: 31
+tags: [PS-Crystal, manga]
+---
+species: Meganium
+setname: PS-Crystal
+ingamename: Megaree
+displayname: Megaree
+ability: [Overgrow]
+item: [null]
+ball: [Poke]
+moves:
+- [Razor Leaf]
+- [Tackle]
+- [Frenzy Plant]
+gender: [m]
+happiness: 255
+biddable: true
+rarity: 1.0
+nature: Hardy
+evs: {hp: 0, atk: 252, def: 0, spA: 4, spD: 0, spe: 252}
+ivs: 31
+tags: [PS-Crystal, manga]
+---
+species: Mr. Mime
+setname: PS-Crystal
+ingamename: Mymee
+displayname: Mymee
+ability: [Soundproof]
+item: [null]
+ball: [Poke]
+moves:
+- [Barrier (=3)]
```

Comment:
I also appreciate you curtailing the PP on this just because it could cheese a lockout.